### DISCUSSION TITLE
How to disable recursive filter?

### DISCUSSION THREAD

**Discussion by schuebel** [nvim-neo-tree/neo-tree.nvim#1296] [Q&A] [answered]
I have a directory structure similar to this:

```
├── a
│   ├── dir
│   │   └── asdf
│   │       └── code
│   ├── dir1
│   │   └── asdf
│   │       └── code
│   └── dir2
│       └── asdf
│           └── code
├── b
├── c
├── code
├── d
├── e
├── f
└── g
    └── a
        └── b
            └── c
                └── d
                    └── code
```

Now when I filter (using `/` or `D`) for _code_ it displays all (sub)directories named _code_. This makes it impossible to quickly move the cursor to the top level directory _code_ because the subdirectories matching the search clutter the filtered results.

Is there a way to search/filter nonrecursively? `help neotree` does not mention anything as far as I understand.

**Comment by bew** [comment, marked as answer]
Does that help?
https://github.com/nvim-neo-tree/neo-tree.nvim/discussions/1242

**Reply by schuebel** [reply]
It absolutely does. Thanks!
